### DISCUSSION TITLE
nzbget-ng

### DISCUSSION THREAD

**Discussion by isarrider** [nzbgetcom/nzbget#37] [General]
Hi,

I guess you are aware of this: https://github.com/nzbget-ng/nzbget
I am not sure Paul ist aware of that here...
Will you cherrypick pullRQ from there?
Or do they have to be there and here?

BR,
Alex

**Comment by luckedea** [comment]
Hello @isarrider 
Yes! I'm looking at all issues and PRs in the original repo, already merged some for this v22 release, but first priority was to make sure project stays alive and builds for all modern OSes, including Apple M1, doesn't break on Windows.

Paul's work for WtF support you mean? There's an extension script for that, trying to respect @hugbug 's vision for that particular PR.

I'm really open for feedback on which Issues or PRs from original repository or -ng should be picked up next.

**Reply by isarrider** [reply]
No, actually I put in a PR but somebody improved my PR then - have a look:
https://github.com/nzbget-ng/nzbget/pull/12 replaced by
https://github.com/nzbget-ng/nzbget/pull/13
It is about the cache size which makes on some (mostly weaker cores a difference)...

Btw awesome that SB keeps this "alive"

**Reply by luckedea** [reply]
Oh you mean merged or closed PRs on -ng.
This wasn't on radar and it is now.

If you'll post an issue or PR here - it will get attention sooner.
If you wont' do that - I'll look into it later anyway.

> Btw awesome that SB keeps this "alive"
It's a great piece of engineering people use.

**Reply by isarrider** [reply]
Well, I would but my PR is superseeded and I dont wanna steal it...
As you got already your releases going, maybe Paul wants to play along here too.
Lets see, Ill keep an eye on it!

**Reply by luckedea** [reply]
https://github.com/nzbgetcom/nzbget/pull/52 fairly straightforward!

**Comment by isarrider** [comment]
Ah and I posted bc I wasn't aware of this here till today as it doesn't appear in forks in nzbget:
https://github.com/nzbget/nzbget/network

**Comment by ghost** [comment]
Why don't you guys combine efforts and produce one NZBGet version. Now there's 2 competing ones which seems pointless.

**Comment by luckedea** [comment]
@Fossil01 Yes absolutely.
Also replied here https://github.com/nzbgetcom/nzbget/issues/83
After v23 (extensions rework) we'll look into it again.

Additionally Paul (author of `nzbget-ng`) kindly offered to route a few PRs & Issues to us which would help a lot.

**Comment by luckedea** [comment]
As we started combining efforts between `nzbgetcom` and `nzbget-ng` it's now matter of going through all issues and accomplishments of `nzbget-ng` and making sure they get into all the builds we do as `nzbgetcom`. I'm going to close this issue to avoid confusion and to free up space for new issues :)
